app.js

```diff
@@ -52,7 +52,7 @@ app.use(
     cookie: {
       sameSite: true,
       httpOnly: true,
-      maxAge: 60000,
+      maxAge: 120000,
```

Comment:
Maybe best to set it for a whole year? `365 * 24 * 60 * 60 * 1000`
Or did you set it to this deliberately ?

Reply:
no deliberate reason, 30 min sounded reasonable but can increase.